Tests/test_ringbuffer.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+
+from ringbuffer import RingBuffer
+
+
+class TestRingBuffer(unittest.TestCase):
+
+    def test_fills(self):
+        r = RingBuffer(10)
```

Comment:
not sure you need to test the ringbuffer class

Reply:
Eh test's written.  I also wrote this after discovering a big memory safety issue in the ringbuffer I was previously using, which seems to have come about from a lack of testing.